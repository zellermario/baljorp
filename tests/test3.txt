create SJ m1 0
create ISJ m2 0 1
addNeighbor m1 1 m2
addNeighbor m2 2 m1
create Eszkimo e1 m1
create Eszkimo e2 m2
move e1 1
pass e1