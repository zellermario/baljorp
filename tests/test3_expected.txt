Fields:
SJ m1 (0) - object: x
	neighbours: [(1,m2)]
ISJ m2 (0,1) - object: x
	neighbours: [(2,m1)]

Buildings:

Actors:
> Eszkimo e1 (heat: 0) on m2
  Inventory: []
> Eszkimo e2 (heat: 0) on m2
  Inventory: []